javatools/src/main/java/org/xvm/tool/Compiler.java

```diff
@@ -98,221 +102,203 @@
  * <li>{@code -v} / {@code --verbose} - provide information about the work being done by the compilation process</li>
  * </ul>
  */
-public class Compiler
-        extends Launcher {
-    /**
-     * Entry point from the OS.
-     *
-     * @param asArg command line arguments
-     */
-    public static void main(String[] asArg) {
-        try {
-            System.exit(launch(asArg));
-        } catch (LauncherException e) {
-            System.exit(e.error ? 1 : 0);
-        }
+public class Compiler extends Launcher<CompilerOptions> {
+
+    protected enum Strictness {
+        None,
+        Suppressed,
+        Normal,
+        Stickler
     }
 
-    /**
-     * Helper method for external launchers.
+    protected Strictness strictLevel = Strictness.Normal;
 
-     * @param asArg  command line arguments
-     *
-     * @return the result of the {@link #process()} call
+    protected ModuleRepository repoLib;
```

Comment:
What is a reason to move fields declarations from the section at the end of the file that says "---- fields ----" under a section in the top of the file that says "----- constants -----"?

I'm fine with fields declared at the top or at the bottom, but randomly changing this in one file and under an incorrect section header seems very strange. Besides, it makes the work of reviewer dramatically harder. Are there any mo-trivial changes here? I cannot say without spending a lot of time unnecessraly